Hunting Queries/OracleDatabaseAudit/OracleDBAuditInactiveUsers.yaml

```diff
@@ -0,0 +1,36 @@
+id: 2c7bbbe1-17cb-44ec-be63-fbb37b609403
+name: OracleDBAudit - Inactive Users
+description: |
+  'Query for searching user accounts which last activity was more than 60 days ago.'
+requiredDataConnectors:
+  - connectorId: OracleDatabaseAudit
+    dataTypes:
+      - Syslog
+tactics:
+  - InitialAccess
+query: |
+  let lbtime_60d = 60d;

```

Comment:
30d max recommended as a default for hunting queries, but not required.  Just remember this could cause performance issues.

Reply:
updated

Reply:
looks like this did not get updated, if you want to leave at 60d, just respond with that. :)

Reply:
fixed